typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -415,6 +415,60 @@ export const gnosis: ChainMetadata = {
   ],
 };
 
+export const inevm: ChainMetadata = {
+  blockExplorers: [
+    {
+      apiUrl: 'https://inevm.calderaexplorer.xyz/api',
+      family: ExplorerFamily.Blockscout,
+      name: 'Caldera inEVM Explorer',
+      url: 'https://inevm.calderaexplorer.xyz/',
+    },
+  ],
+  blocks: {
+    confirmations: 1,
+    estimateBlockTime: 3,
+    reorgPeriod: 0,
+  },
+  chainId: 2525,
+  domainId: 2525,
+  displayName: 'Injective EVM',
+  displayNameShort: 'inEVM',
+  name: Chains.inevm,
+  nativeToken: {
+    decimals: 18,
+    name: 'Injective',
+    symbol: 'INJ',
+  },
+  protocol: ProtocolType.Ethereum,
+  rpcUrls: [{ http: 'https://inevm.calderachain.xyz/http' }],
```

Comment:
I recommend adding more RPCs to take advantage of SmartProvider's fallback:
https://github.com/cosmos/chain-registry/blob/master/injective/chain.json#L217